core/src/main/scala/kafka/tools/ImportZkOffsets.scala

```diff
@@ -77,21 +77,24 @@ object ImportZkOffsets extends Logging {
   }
 
   private def getPartitionOffsetsFromFile(filename: String):Map[String,String] = {
-    val fr = new FileReader(filename)
-    val br = new BufferedReader(fr)
-    var partOffsetsMap: Map[String,String] = Map()
-    
-    var s: String = br.readLine()
-    while ( s != null && s.length() >= 1) {
-      val tokens = s.split(":")
-      
-      partOffsetsMap += tokens(0) -> tokens(1)
-      debug("adding node path [" + s + "]")
-      
-      s = br.readLine()
+    val br = new BufferedReader(new FileReader(filename))
```

Comment:
`Source` is a class designed to read source files and it had a few pitfalls so the recommendation is generally to stay away (see http://stackoverflow.com/questions/4458864/whats-the-right-way-to-use-scala-io-source for one example).